deploy/lib/application-stack.ts

```diff
@@ -88,6 +94,10 @@ export class ApplicationStack extends Stack {
           value: bucket.bucketName,
           type: BuildEnvironmentVariableType.PLAINTEXT,
         },
+        VITE_CLOUDFRONT_DISTRIBUTION_ID: {
```

Comment:
The CF Distribution Id doesn't need to be baked into the React App (The `VITE_` prefix) right?

Reply:
yes, as it will be import to Vite code, we can remove it ...